callmetadata.schema.json

```diff
@@ -5,29 +5,87 @@
     "description": "Meta data supporting dialtone, ringdown, and shoutdown calls",
     "type": "object",
     "definitions": {
-        "participant": {
+        "states": {
             "type": "object",
-            "description": "Specific details about the farEndUsers (the other end of the call)",
             "properties": {
-                "firmName": {
+                "state": {
                     "type": "string",
-                    "example": "XYZ Firm",
-                    "description": "Name of the farEnd firm on the call. If an internal call, the firmName can match the nearEnd firm"
+                    "description": "name of the latest state of the current communication: CallOriginated, CallDelivered, CallEstablished, ect. Could be an enum with defined states. ",
+                    "example": "CallDelivered"
                 },
-                "groupId": {
+                "time": {
+                    "type": "string",
+                    "format": "date-time",
+                    "description": "Date and time when the state changed for the given participant",
+                    "example": "2016-06-18T08:05:15Z"
+                },
+                "duration": {
                     "type": "integer",
-                    "example": 235,
-                    "description": "The internal ID used by consumer for the farEnd group. This will never change for the connection"
+                    "description": "Time in milliseconds how of how long the call has been in previous states for the given participant",
+                    "example": 27978
+                }
+            }
+        },
+        "member": {
+            "name": {
+                "type": "string",
+                "example": "Steve Jones",
+                "description": "Name of the participant communicating as defined in schema.org"
+            },
+            "groupName": {
+                "type": "string",
+                "example": "Energy Desk",
+                "description": "Name of the group for which the participant is representing in the communication"
+            },
+            "firmName": {
+                "type": "string",
+                "example": "XYZ Firm",
+                "description": "Name of the participant's firm represnted in the communication"
+            },
+            "timeZoneOffset": {
+                "type": "string",
+                "example": "UTC -05:00:00",
+                "description": "The UTC timezone offset for the participant"
+            }
+        },
+        "participant": {
+            "type": "object",
+            "description": "Specific details about a participant in a given communication",
+            "properties": {
+                "member": {
+                    "$ref": "#/definitions/member"
                 },
-                "groupName": {
+                "involvement": {
```

Comment:
@pinczo Great addition. Since there's an open PR related to participants I'll add this change in the early opened PR.

Here is the commit with your username as the author:

https://github.com/finos/voice-metadata-standard/pull/44/commits/0c9e0ecaec3472133f1c08c9e17a3be1e828805c